odl/solvers/functional/functional.py

```diff
@@ -0,0 +1,1160 @@
+        """
+        return proximal_quadratic_perturbation(
+            self.orig_cconj_f.proximal, a=0, u=self.translation)
+
+    @property
+    def convex_conj(self):
+        """Convex conjugate functional of the functional.
+
+        By the Fenchel-Moreau theorem this a translation of the original
+        convex conjugate functional.
+        """
+        return self.orig_cconj_f.convex_conj.traslated(
+            self.translation)
+
+
+class ConvexConjugateFuncScaling(Functional):
+
+    """ The ``Functional`` representing ``(scaling * F(.))^*``.
+
+    This is a functional representing the conjugate functional of the scaled
+    function ``scaling * F(.)``. This is calculated according to the rule
+
+        ``(scaling * F(.))^* (x) == scaling * F^*(x/scaling)``,
+
+    where ``scaling`` is the scaling parameter. Note that scaling is only
+    allowed with strictly positive scaling parameters.
+
+    Notes
+    -----
+    The implementation assumes that the underlying  functional ``F`` is proper,
+    convex, and lower semi-continuous. Otherwise the convex conjugate of the
+    convex conjugate is not necessarily the functional itself.
+
+    For reference on the identity used, see [KP2015]_.
+    """
+
+    def __init__(self, cconj_f, scaling):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        cconj_f : `Functional`
+            Functional corresponding to ``F^*``.
+        scaling : `float`, positive
+            Positive scaling parameter.
+        """
+        if not isinstance(cconj_f, Functional):
+            raise TypeError('`cconj_f` {} is not a `Functional` instance'
+                            ''.format(cconj_f))
+
+        scaling = float(scaling)
+
+        # The case scaling = 0 is handeled in Functional.__rmul__
+        if scaling <= 0:
+            raise ValueError(
+                'Scaling with nonpositive values is not allowed. Current '
+                'value: {}.'.format(scaling))
+
+        super().__init__(space=cconj_f.domain, linear=cconj_f.is_linear,
+                         grad_lipschitz=(np.abs(scaling) *
+                                         cconj_f.grad_lipschitz))
+
+        self.__orig_cconj_f = cconj_f
+        self.__scaling = scaling
+
+    @property
+    def orig_cconj_f(self):
+        """The original convex conjugate functional."""
+        return self.__orig_cconj_f
+
+    @property
+    def scaling(self):
+        """The scaling."""
+        return self.__scaling
+
+    def _call(self, x):
+        """Apply the functional to the given point."""
+        return self.scaling * self.orig_cconj_f(x * (1.0 / self.scaling))
+
+    @property
+    def gradient(self):
+        """Gradient operator of the functional."""
+        return self.orig_cconj_f.gradient * (1.0 / self.scaling)
+
+    @property
+    def proximal(self):
+        """Return the proximal factory of the ConvexConjugateFuncScaling.
+
+        See Also
```

Comment:
This is somewhat contradictory to the paradigm that properties don't *return* (or *do* anything), but rather *are*. I'm not totally against having a clarifying `Returns` section, but we should make a conscious decision here. 
For the summary line, it's clear that there should not be a "Return" for properties.